.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: pymongoexplain
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
Should this be called "Test" instead of "Build"?

Reply:
Done.